spec/components/provider_interface/interview_and_course_summary_component_spec.rb

```diff
@@ -24,6 +24,16 @@
     expect(component).to include(interview.location)
   end
 
+  it 'applies a hidden field to the change link' do
+    component = render_inline(described_class.new(interview: interview, user_can_change_interview: true))
+    expect(component.css('.govuk-visually-hidden').first.text).to include(interview.date_and_time.to_s(:govuk_date_and_time))
+  end
+
+  it 'applies a hidden field to the cancel link' do
```

Comment:
If we are to test this our spec should indicate why this is important.

```suggestion
  it 'cancel link has a hidden field to improve accessibility for visually impaired users' do
```